templates/download/mediatek.html

```diff
@@ -0,0 +1,109 @@
+{% extends "download/_base_download.html" %}
+
+{% block title %}Install Ubuntu on MediaTek{% endblock %}
+
+{% block meta_description %}Use Ubuntu on MediaTek for the familiar developer experience and an accelerated path to production {% endblock meta_description %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1RMvAI7Soa_zIlP0e63BMcFT3DJfvLrMS2EyiOvcy-xc/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-topped is-bordered">
+  <div class="row">
+    <div class="col-7">
+      <h1>Install Ubuntu on MediaTek</h1>
+      <p>Run Ubuntu on your MediaTek Genio 1200. </p>
+      <p>Pick the OS image, flash it onto internal EMMC, and away you go.</p>
+      <p><a href="/download/contact-us" class="js-invoke-modal p-button--positive">Get in touch</a></p>
+    </div>
+    <div class="col-5 u-vertically-center u-align--center u-hide--small u-hide--medium">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/0ebb345c-MediaTek_logo.svg",
+        alt="",
+        width="1024",
+        height="268",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-7">
+      <h2>Download Ubuntu Server</h2>
+      <h3 class="p-heading--4">Ubuntu Server 22.04 LTS</h4>
+      <p>The version of Ubuntu with up to 10 years of long term support, until April 2032.</p>
+      <p><a href="#" class="p-button--positive">Download 64-bit</a></p>
+      <p>Works on:</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">MediaTek Genio 1200</li>
+      </ul>
+    </div>
+    <div class="col-5 u-vertically-center u-align--center u-hide--small u-hide--medium">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/aa889cf5-MediaTek_Genio+1200_Back_NOBG_Shadow_0422.png",
+        alt="",
+        width="3840",
+        height="2160",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <h3 class="p-heading--4">First time installing Ubuntu on MediaTek?</h3>
+    <p>Please check the <a href="https://www.mediatek.com/aiot/products/mediatek-genio/1200">MediaTek Genio page</a> for more information.</p>
```

Comment:
The link works but there's nothing in the page...
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/57550290/196173434-b336bda0-7c38-4ec6-97d7-a62cd68fe5ae.png">